exceptions.txt

```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers (2015)', 'Ballers (2014)',
+281708: 'Proof', 'Proof (US)',
+294417: 'Scream The TV Series',
```

Comment:
![image](https://cloud.githubusercontent.com/assets/390379/8716249/9427e7e8-2b43-11e5-8744-891f7c987082.png)
The scream exception is not needed, thetvdb already has this exception, so adding it here is a double excpetion.